### ISSUE TITLE
[Bug]: MudExChipSelect focus issue

### ISSUE THREAD

**Issue by jaypalexa** [fgilde/MudBlazor.Extensions#34]
### What happened?

We have run into problems being able to consistently be able to get focus into the Filter textbox when using MudExChipSelect.

Your response from another (unrelated) [issue](https://github.com/fgilde/MudBlazor.Extensions/issues/32):
> I will have a look at it. I know MudBlazor have had a bug and it was solved. You can see it here its working https://cae4f752.mudblazor-extensions.pages.dev/chip-select but now it looks like the Bug is back and now here its no more working https://www.mudex.org/chip-select. But I didnt changed anything in MudExChipSelect or dependent logic.
> Maybe I need to rebuild the chipselect. However can I then close this issue? For the chip select you can create a new one.

### Expected Behavior

Focus is given to the Filter textbox when clicking on it

### What browser are you using?

Chrome

### Sample Solution

[TestApp.zip](https://github.com/fgilde/MudBlazor.Extensions/files/12239282/TestApp.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
I work on it but today I start in a 10 days holiday travel. I continue afterwards

**Comment by fgilde** [comment]
Just as info I work on it, but it's very frustrating and tricky, because all things on mud side are internal or private. So I create a new MudExSelect and inherit it.
Looks good in general but makes currently problems in the MudExObjectEdit.

**Comment by fgilde** [comment]
Fixed with [1.7.65 Nuget](https://www.nuget.org/packages/MudBlazor.Extensions/1.7.65#readme-body-tab)

**Comment by jaypalexa** [comment]
Thank you, Florian. Took me awhile to get back to this, but I wanted to let you know that we finally implemented your MudExSelect refactor and everything seems to be working great. We replaced our usage of the Codebeam MudSelectExtended with your MudExSelect, and now we are able to remove Codebeam from the solution entirely. Glad to use one fewer (and possibly conflicting) third-party library. Thanks again for all of your hard work and dedication and attention to MudEx.
